src/Datadog.Trace/Agent/TransportStrategy.cs

```diff
@@ -0,0 +1,31 @@
+using Datadog.Trace.Agent.StreamFactories;
+using Datadog.Trace.Agent.Transports;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.HttpOverStreams;
+using Datadog.Trace.Logging;
+
+namespace Datadog.Trace.Agent
+{
+    internal static class TransportStrategy
+    {
+        private static readonly Vendors.Serilog.ILogger Log = DatadogLogging.For<Tracer>();
+
+        public static IApiRequestFactory Get(TracerSettings settings)
+        {
+            var strategy = settings.TraceTransport?.ToUpperInvariant();
+
+            switch (strategy)
+            {
+                case "DATADOG-TCP":
+                    Log.Information("Using {0} for trace transport.", nameof(TcpStreamFactory));
+                    return new HttpStreamRequestFactory(new TcpStreamFactory(settings.AgentUri.Host, settings.AgentUri.Port), new DatadogHttpClient());
+                case "DATADOG-NAMED-PIPES":
+                    Log.Information("Using {0} for trace transport, with pipe name {1} and timeout {2}.", nameof(NamedPipeClientStreamFactory), settings.TracesPipeName, settings.TracesPipeTimeoutMs);
```

Comment:
```suggestion
                    Log.Information("Using {0} for trace transport, with pipe name {1} and timeout {2}ms.", nameof(NamedPipeClientStreamFactory), settings.TracesPipeName, settings.TracesPipeTimeoutMs);
```

Reply:
This and other comments are marked as resolved without the corresponding code changes. Did you make the changes locally and haven't pushed yet?

Reply:
Never mind. May have been a Github UI thing showing me outdated code.

Reply:
I made these changes and pushed them.